ParabolInc/parabol
# Fix #3753 comment count in retro summary

Fixes #3753 

### Tests
- [ ] The title, votes, and comment link are all clickable; clicking takes you to the discussion
- [ ] Depending on count, the comment link label is:
  - No Comments (neutral gray link instead of blue)
  - See 1 Comment
  - See N Comments
  - See 100+ Comments (I’m not sure we need to query for many comments so I capped it at 100.)
- [ ] This works in both the email and browser views
- [ ] The demo summary has static topic titles and votes (no links back to the comments)

### Note
Doesn’t solve for bugs (re)discovered that exist in prod:
- The email topic links are busted with utm params before the `/discuss/N` sub path of the URL, see #3779
- The timeline shows meetings that were ended early, the summary link doesn’t work, see #3778
